script.js

```diff
@@ -0,0 +1,114 @@
+
+//The display button on click of which the calendar will be displayed
+let buttonD = document.querySelector(".display-button");
```

Comment:
A next step to this app can be removing this display button and adding a feature - on change of month and year the calendar can automatically be re-rendered. Also on first init the calendar can start from currentDate